.github/workflows/publish.yml

```diff
@@ -100,9 +100,10 @@ jobs:
       - name: Build image
         run: |
           cd $( dirname ${{ matrix.dockerfile.path }})
+          sed -i '/jupyterlab/d' requirements.txt
           VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
           sed -i "s#etna\[all\]#etna\[all\]==$VERSION#g" requirements.txt
-          poetry export --without-hashes --format=requirements.txt >> requirements.txt
+          poetry export -E "all jupyter" --without-hashes --format=requirements.txt >> requirements.txt
```

Comment:
Don't we want to delete requirements.txt?

Reply:
If we remove `requirements.txt`, etna library won't be installed. During export of dependencies it only exports dependencies, not the package itself and we should install the package too.

Reply:
Ok, then better to append `etna` here

Current behaviour is not straightforward - we have requirements.txt which is not actual.

The best way is to create multi-stage build to generate requirements but it could be not trivial

Reply:
Currently creating `requirements.txt` consists of steps:
- Take existing `requirements.txt`
- Remove `jupyterlab` dependency
- Replace generic etna dependency specific etna dependency that points to relevant for CI version
- Append into result `requirements.txt` dependencies from `poetry.lock`

What kind of changes are you suggesting?